gumby/modules/base_ipv8_module.py

```diff
@@ -96,14 +96,12 @@ def setup_config(self):
         from ipv8 import community
         community._DEFAULT_ADDRESSES = []
         community._DNS_ADDRESSES = []
-        head_host = gethostbyname(environ['HEAD_HOST']) if 'HEAD_HOST' in environ else "127.0.0.1"
+        head_host = gethostbyname(os.environ['HEAD_HOST']) if 'HEAD_HOST' in os.environ else "127.0.0.1"
 
-        my_state_path = path.abspath(path.join(environ["OUTPUT_DIR"], ".%s-%d-%d" % (self.__class__.__name__,
-                                                                                     getpid(), self.my_id)))
-        makedirs(my_state_path)
-        bootstrap_file = path.join(environ['OUTPUT_DIR'], 'bootstraptribler.txt')
-        if path.exists(bootstrap_file):
-            symlink(bootstrap_file, path.join(my_state_path, 'bootstraptribler.txt'))
+        my_state_path = os.getcwd()
```

Comment:
`os.getcwd()` is not a stable path, it can change at any moment by anyone and anything.

I'd prefer anchoring folders from stable points like `os.environ['OUTPUT_DIR']` or even the top-level `run.__file__` file.

Reply:
That we change current working directory is actually a feature of Tribler. Specifically, the moment that we receive a peer ID from the central experiment server, we change the current working directory to a directory with the peer ID in there.

I'll try to think of a different solution for this.

Reply:
I have implemented an alternative solution, where I explicitly set the right directory.